core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
OK, already started on that, will revert my changes.

On Fri, Dec 18, 2015 at 5:21 PM, Bert Roos <notifications@github.com> wrote:

> In
> core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java
> <https://github.com/yonadev/yona-server/pull/63#discussion_r48041649>:
>
> > @@ -5,21 +5,16 @@
> >  package nu.yona.server.messaging.entities;
>
> I understand caching all messages is too much of a leap. Please revert
> MessageDestination to the original design. We might need a discussion in
> front of a whiteboard to conclude on what change is needed here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yonadev/yona-server/pull/63/files#r48041649>.
>